### ISSUE TITLE
A new install issue

### ISSUE THREAD

**Issue by SiccarPoint** [landlab/landlab#349]
Just worked through a new strange path issue for installs on Kelly Kochanski's machine.

Turns out that if an install is performed on a Windows machine where conda is installed *through cygwin*, some very strange errors can happen. Kelly was running Python 3, but I don't think that's relevant. In either a Command Prompt iPython session or a Spyder session, landlab will import cleanly, but `landlab.test()`will result in hundreds (~500) of fails. Closer inspection will show many of these appear to be related to decorators, and crucially _are not Landlab fails_ - they're triggered in the unit tests of other python modules. `where python` and `where ipython` both look to the same user Anaconda install.

15 minutes of work on this seemed to suggest it's a subtle versioning issue, where more than one Python install is somehow present, or at least, the conda Python loses track of some of its own components. I was able to confirm that *from a cygwin window*, a more conventional versioning issue can be observed - `which python` looks at the system python, and `which ipython` looks at `cygwin/user/[...]/Anaconda3/...`. BUT this ipython is in fact the one true python version, and is the only place to which landlab is properly bound. It is not the same python version as that seen from the command prompt and the Spyder dynamic window, much as it might seem to be.

TL/DR:
* On windows, installing conda and/or landlab though the cygwin window can result in a world of pain.
* Temporary workaround: Run landlab exclusively through a cygwin ipython session.
* Actual fix: ???

**Comment by SiccarPoint** [comment]
As our resident PC guru, @saisiddu 's thoughts would be very welcome here. Ever seen an issue like this, Sai?
